assets/src/application/ui/input/Text.tsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import { Input } from 'antd';
+import { InputProps } from 'antd/lib/input';
 
-const Text = ({ input, htmlType = 'text', ...rest }) => {
-	return <Input {...input} {...rest} type={htmlType} />;
```

Comment:
Here as well, `input` object must be passed to the input.

Reply:
I don't understand what you mean, `props` includes `input` prop, why this needs to be passed separately and the spreaded each individually.

Reply:
this was clarified in the chat, thx